src/main/java/gov/nasa/pds/label/Label.java

```diff
@@ -647,7 +648,12 @@ private DataObject makeTable(gov.nasa.arc.pds.xml.generated.File file, TableDeli
       offset = table.getOffset().getValue().longValueExact();
     }
     long size = -1;
-    if (file.getFileSize() != null) {
+    if (table.getRecordDelimited() != null) {
+      RecordDelimited definition = table.getRecordDelimited();
+      if (definition.getMaximumRecordLength() != null) {
```

Comment:
@jordanpadams 

I would suggest keeping the check. Yes, if the max is not defined, maybe it should return something other than -1 which will always succeed like something to force it to fail but we can always compute the minimal location to make sure there is no overlap. Even with the max defined, it only says where the table may end and does not include potential blanks. It does become more erroneous the more blank areas one wants to skip around, but it will, at least, catch the most egregious errors.

Reply:
@al-niessner my concern here is for the Table_Delimited that contains records with lengths < the `maximum_record_length`

For example, I am pretty sure [this label example](https://github.com/NASA-PDS/pds4-jparser/files/11117001/mre_cal_sc_ttcp_delay_schulte_01s_2021069_fail.xml.txt) is valid. It basically shows what if we have a Table_Delimited with 2 records and max record length 294 bytes, but what if 1 of those records is half that size?